### PR TITLE
Make package ready for PyPI

### PR DESCRIPTION
**Pull request by FrancescoCasalegno**
## Description

This PR introduces the following changes, finalized to the release of `atlalign` on PyPI.

1. Added some extra fields in `setup.py` to make our package easier to find on PyPI.
1. Removed `extra_requires.tf` and merged it with `install_requires`.
1. Removed `keras_contrib` from dependencies (was not used)
1. Removed `lpips_tf` from dependencies (it is not on PyPI so it made pushing `atlalign` to PyPI impossible)
1. If a user wants to use functionalities of `atlalign` relying on `lpips_tf`(namely: losses or metrics), an Exception with instructions on how to install it is raised. See "Testing this PR".
1. Made Tox install `lpips_tf` before running tests that require it.
1. Updated information on installing in `README.md` 
   - removed `[tf]`
   - added info about installing from PyPI
   - added info about installing extra dependency LPIPS-TF

## Testing this PR
To test this PR you can run
```python
>>> from atlalign.metrics import demons_img
...
ModuleNotFoundError: 
        LPIPS-TensorFlow required but not found.

        Please install it by running the following command:
        $ pip install git+http://github.com/alexlee-gk/lpips-tensorflow.git#egg=lpips_tf
```
or
```python
>>> from atlalign.ml_utils.losses import jacobian_distance
...
ModuleNotFoundError: 
        LPIPS-TensorFlow required but not found.

        Please install it by running the following cmmand:
        $ pip install git+http://github.com/alexlee-gk/lpips-tensorflow.git#egg=lpips_tf

```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,39 +54,51 @@ brain atlases obtained with ISH gene expression and Nissl stains.
...
 
-```bash
+#### Installation Requirements
+
+##### Pip
+First of all, make sure the `pip` version is at least `19.1`.
+```shell script
 pip install --upgrade pip
 pip --version  # make sure >= 19.1
 ```
```
Comment (jankrepl) [inline README.md:71]: I think this `pip` update is not necessary anymore because we are dropping the `@` syntax for direct links

**Review by jankrepl — COMMENTED** [review]
Great job! 

I have one remark. The docs also have a section on installation, maybe we should update it too.

File: README.md (hunk 1 above)
Comment (jankrepl) [reply README.md:71]: Sure, having a recent `pip` is always a good thing. However, IMO it is not necessary to install the package.

Comment (FrancescoCasalegno) [reply README.md:71]: Don't we use it in `tox.ini` at line `13`?
```
    lpips_tf @ git+http://github.com/alexlee-gk/lpips-tensorflow.git#egg=lpips_tf
```

Comment (jankrepl) [reply README.md:71]: Hmmm. The thing is that `tox` will have it's own `pip` and it is not going to be the one that we instruct the user to update in the README (the one that lies in our virtual environment).

See https://github.com/tox-dev/tox/issues/791 for reference.

Comment (FrancescoCasalegno) [reply README.md]: Fixed this + docs in 10aa518

**Review by jankrepl — APPROVED** [review]

**Pull request merged**